crates/typos-dict/assets/words.csv

```diff
@@ -1,32 +1,32 @@
-aaccess,access
-aaccessibility,accessibility
-aaccession,accession
+aaccess
+aaccessibility
+aaccession
 aache,ache,cache
-aack,ack
-aactual,actual
-aactually,actually
-aadd,add
-aadded,added
-aadding,adding
-aagain,again
-aaggregation,aggregation
-aand,and
-aanother,another
-aapply,apply
+aack
```

Comment:
The redundant a's is a separate thing and we should be correcting these

The challenge with blindly checking concatenated words is it doesn't filter out for when they don't make sense.

Reply:
In general, I'd feel better if we just looked at what changed due to  the spaces and applied it to those.  We could then separate decide which of these changes might make sense.  As is, I'm seeing a lot that don't and don't want to take the time to decide that.

Reply:
Oh the logic I had did actually already detect these with

```rs
            if only_correction.starts_with(prefix) {
                return false;
            }
```

I just must have omitted resetting `words.csv` to run `SNAPSHOTS=overwrite cargo test verify` again.

Reply:
So the many cases have just become:

```
aequidistant
aequivalent
afor
amuch
anumber
ascripts
asudo
imakes
isimilar
itheir
itheirs
iwithout
```

which I think make sense to not correct automatically.

Reply:
`iwithout` and many of these don't make sense from a "word combining" perspective.  Things we can correct in master will become uncorrectable with this change.